### DISCUSSION TITLE
Non-stationary kriging factors?

### DISCUSSION THREAD

**Discussion by RyanConway91** [pypest/pyemu#472] [Q&A]
I would like to develop a k field with anisotropy that follows a meandering buried alluvial channel, and preferably calibrate the degree of and anisotropy and the bearing. Is there any way to do non-stationary kriging in pyemu/PEST? Or would the approach be to be krige the channel in little chunks and combine the data together in the end?

**Comment by rhugman** [comment]
Checkout 'pypestutils' (integratable with pyemu; see examples in the repo) and/or J.Doherty's 'alluvsim' utility (pesthomepage.org). Example applications of the latter in one of the first GMDSI worked examples.
